addons/l10n_tr/__manifest__.py

```diff
@@ -4,20 +4,25 @@
     'website': 'https://www.odoo.com/documentation/17.0/applications/finance/fiscal_localizations.html',
     'icon': '/account/static/description/l10n.png',
     'countries': ['tr'],
-    'version': '1.1',
+    'version': '1.2',
     'category': 'Accounting/Localizations/Account Charts',
     'description': """
-Türkiye için Tek düzen hesap planı şablonu Odoo Modülü.
-==========================================================
+    This was done in collaboration with Broadmax Partner in Turkey.
 
-Bu modül kurulduktan sonra, Muhasebe yapılandırma sihirbazı çalışır
-    * Sihirbaz sizden hesap planı şablonu, planın kurulacağı şirket, banka hesap
-      bilgileriniz, ilgili para birimi gibi bilgiler isteyecek.
+    Uniform chart of accounts template Odoo Module for Türkiye.
+    ============================================================================== ========
+
+    Once this module is installed, the Accounting configuration wizard runs
+        * The wizard will ask you for the chart of accounts template, the company where the plan will be established, and the bank account.
+        It will ask for information such as your information, the relevant currency.
```

Comment:
It doesn't mean anything :sob:  What about something more generic:
```
Turkish charts of accounts
========================================

    * Defines the default chart of accounts
    * Defines the default taxes
    * Defines default tax report

This was done in collaboration with Broadmax Partner in Turkey.
""",
```
Don't forget to add it to `base.pot` for translations

Reply:
Done! first time to see this `base.pot` file tho.
I think this PR is cursed NGL :rofl: